packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -158,6 +170,16 @@ const Set<PointerDeviceKind> _kLongPressSelectionDevices = <PointerDeviceKind>{
 /// child selection area can not extend past its subtree, and the selection of
 /// the parent selection area can not extend inside the child selection area.
 ///
+/// ## Tests
+///
+/// In a test, a region can be selected either by faking drag events (e.g. using
+/// [WidgetTester.dragFrom]) or by sending intents to a widget inside the region
+/// that has been given a [GlobalKey], e.g.:
+///
+/// ```dart
```

Comment:
\`\`\`dart sections do get analyzed.

Reply:
I am ... surprised. I thought we only analyze stuff in `{@tool ...}` - I actually have it on my list to look into what to do with raw ```dart sections... Do you know what script/CI analyzed this for you?

Reply:
oh i see, even the \`\`\`dart sections need {@tool snippet}, i hadn't realized that

Reply:
must have been a different block that caught that error

Reply:
I don't want the @tool section because that makes a giant blue block which would break the flow...

Reply:
I do agree that the blue box can be disturbing, but I still would prefer code that continues to function (because it is analyzed). Maybe we need to implement a lightweight `{@tool ...}` variant that doesn't provide a lot of UI chrome, but still analyzes code.

Reply:
https://github.com/flutter/flutter/pull/108500 will make this be analyzed

Reply:
ok this is now analyzed. :-)